README.md

```diff
@@ -176,10 +182,10 @@ Releases are hosted on [jcenter](https://jcenter.bintray.com/com/apollographql/a
 
 Latest development changes are available in Sonatype's snapshots repository:
 
-```
-  repositories {
-    maven { url 'https://oss.sonatype.org/content/repositories/snapshots/' }
-  }
+```gradle
```

Comment:
I never know whether I should use `groovy` or `gradle` here. It looks like Github support both. Do you know any reason to use one over the other?

Reply:
Github supports both not everywhere. I think our docs site may not support it actually

Reply:
For Gradle stuff, especially for simple things like this, I started to put only Kotlin since it also is valid in Groovy

Reply:
If I understand your meaning, I could use `kotlin` as the format flavor?

I found [an example docs page](https://raw.githubusercontent.com/apollographql/apollo-android/master/docs/source/essentials/migration.md) which includes this block:
~~~
```kotlin:title=build.gradle
// Replace:
implementation("com.apollographql.apollo:apollo-android-support:x.y.z")

// With:
implementation("com.apollographql.apollo:apollo-normalized-cache-sqlite:x.y.z") // highlight-line
```
~~~

The rest are ```groovy.

Reply:
Yep, trying to find something that works in both Kotlin and Groovy would be nice. In this case, I believe the below would do?

```
    repositories {
        maven {
            url = uri("https://oss.sonatype.org/content/repositories/snapshots/")
        }
    }
```